### PR TITLE
setup fileshare interface; listed empty methods of AzStorageConnection 

### PR DESCRIPTION
**Pull request by meganliu-msft**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Blobfuse2 - A Microsoft supported Azure Storage FUSE driver
+# Filesfuse - A Microsoft supported Azure Storage FUSE driver
```
Comment (gapra-msft) [inline README.md:1]: Leave this as Blobfuse2 for now

File: component/azstorage/file_share.go [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+	"os"
+
+	"github.com/Azure/azure-storage-file-go/azfile"
+)
+
+type FileShare struct {
+	AzStorageConnection
+	Auth      azAuth
+	Service   azfile.ServiceURL
+	Container azfile.ShareURL
```
Comment (gapra-msft) [inline component/azstorage/file_share.go:48]: name this as Share, just to keep consistent with File Share naming

**Review by gapra-msft — APPROVED** [review]
lgtm!

**Review by tasherif-msft — APPROVED** [review]

File: component/azstorage/file_share.go [hunk 3]
```diff
@@ -0,0 +1,152 @@
+/*
+    _____              _____   _____    _____             _____    _____
+   |       |  |       |       |        |       |     |  |         |
+   |       |  |       |       |        |       |     |  |         |
+   |----   |  |       |----   |-----|  | ----  |     |  |------|  |----
+   |       |  |       |             |  |       |     |         |  |
+   |       |  |_____  |_____   _____|  |       |_____|   ______|  |_____
```
Comment (souravgupta-msft) [inline component/azstorage/file_share.go:7]: Shouldn't this be Blobfuse2 instead of Filesfuse?

**Review by souravgupta-msft — APPROVED** [review]

**Review by vibhansa-msft — APPROVED** [review]

**Pull request merged**